MIGRATIONS.md

```diff
@@ -0,0 +1,32 @@
+# Migrations Document
+
+This document is to outline different migrations needed to update the database as changes are made.  It's an ad hoc solution, and provides more context than actual usefulness.  Migrations should be done in reverse order so the most recent is first, and should include:
+
+* Issue Number
+* Date
+* Overview of change
+* SQL/script statements needed for change
+
+## Issue 192 - Enlace - Absent/Present within attendance - March 2018
+
+Expands the ttm-enlace.Absence table to include an Absent flag so that Enlace can track attendance without a default of attending.  This requires not only a change to the table, but also creation of all rows in the past to explicitly mark as attending.
+
+SQL Needed
+```
+USE ttm-enlace
+ALTER TABLE `Absences` ADD COLUMN `Absent` int(11);
+UPDATE `Absences` SET `Absent` = 1;
+```
```

Comment:
Why use an `int` column instead of a boolean?

Reply:
Made it consistent with booleans elsewhere in the database (Monday in Programs table for instance)